File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
 var runningSum = function(nums) {
  var resultArray = [nums[0], ];
  for (var i = 1; i < nums.length; i++) {
      resultArray.push(nums[i] += nums[i -1]);
  }
  return resultArray;
};

console.log(runningSum([1,2,3,4]));

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        result = []
        running_sum = 0
        for i in nums:
            running_sum += i
            result.append(running_sum)
        return result
